sklearn/model_selection/_split.py

```diff
@@ -634,6 +634,117 @@ def split(self, X, y, labels=None):
         """
         return super(StratifiedKFold, self).split(X, y, labels)
 
+class HomogeneousTimeSeriesCV(_BaseKFold):
+    """Homogeneous Time Series cross-validator
+
+    Provides train/test indices to split time series data in train/test sets.
```

Comment:
Please clarify to the user that we assume samples from later times have higher indices (and thus shuffling in CV is inappropriate).